domain/src/main/java/org/oppia/domain/exploration/ExplorationRetriever.kt

```diff
@@ -23,7 +23,7 @@ class ExplorationRetriever @Inject constructor(
   internal suspend fun loadExploration(explorationId: String): Exploration {
     return when (explorationId) {
       TEST_EXPLORATION_ID_5 -> loadExplorationFromAsset("welcome.json")
-      TEST_EXPLORATION_ID_6 -> loadExplorationFromAsset("about_oppia.json")
```

Comment:
Please introduce a new ID rather than overwriting this--other tests depend on this exploration being about_oppia.

Also, please make sure you're running the domain tests before submitting PRs--I didn't catch this until I started working on tests for #270.

Reply:
reverted TEST_EXPLORATION_ID_6 to `about_oppia.json`

Reply:
Added new Exploration id for ItemSelectionInput.